legion/sdk/legion/sdk/clients/deployment.py

```diff
@@ -179,22 +178,4 @@ def build_client(args: argparse.Namespace = None, retries=3, timeout=10) -> Mode
     :param retries: number of retries
     :param args: (optional) command arguments with .namespace
     """
-    host, token = None, None
-
-    if args:
-        if args.edi:
-            host = args.edi
-
-        if args.token:
-            token = args.token
-
-    if not host or not token:
-        host = host or config.EDI_URL
-        token = token or config.EDI_TOKEN
-
-    if host:
-        client = ModelDeploymentClient(host, token, retries=retries, timeout=timeout)
-    else:
-        raise Exception('EDI endpoint is not configured')
-
-    return client
+    return legion.sdk.clients.edi.build_client(args, retries=retries, timeout=timeout, cls=ModelDeploymentClient)
```

Comment:
the same

Reply:
Commented above - same function name redeclaration